test/message-v2-test.js

```diff
@@ -0,0 +1,666 @@
+			var message = new Message({});
+			var result = message._handleResponse({
+				body       : "<Test>test</Test>",
+				statusCode : 200
+			});
+			var body = result[0];
+			body.Test._text.should.equal("test");
+		})	;
+		it("should handle iris error 1", function () {
+			var message = new Message({});
+			assert.throws(function () {
+				message._handleResponse({
+					body       : "<Response><ErrorCode>Code</ErrorCode><Description>Description</Description></Response>",
+					statusCode : 200
+				});
+			}, "Code: Description");
+		});
+		it("should handle iris error 2", function () {
+			var message = new Message({});
+			assert.throws(function () {
+				message._handleResponse({
+					body       : "<Response><Error><Code>Code</Code><Description>Description</Description></Error></Response>",
+					statusCode : 200
+				});
+			}, "Code: Description");
+		});
+		it("should handle iris error 3", function () {
+			var message = new Message({});
+			assert.throws(function () {
+				message._handleResponse({
+					body       : "<Response><Errors><Code>Code</Code><Description>Description</Description></Errors></Response>",
+					statusCode : 200
+				});
+			}, "Code: Description");
+		});
+		it("should handle iris error 4", function () {
+			var message = new Message({});
+			assert.throws(function () {
+				message._handleResponse({
+					body       : "<Response><resultCode>Code</resultCode><resultMessage>Description</resultMessage></Response>",
+					statusCode : 200
+				});
+			}, "Code: Description");
+		});
+		it("should handle iris undefined error", function () {
+			var message = new Message({});
+			assert.throws(function () {
+				message._handleResponse({
+					body       : "",
+					statusCode : 400
+				});
+			}, "Http code 400");
+		});
+	});
+	describe("makeIrisRequest()", function () {
+		before(function () {
+			nock.disableNetConnect();
+			nock("https://dashboard.bandwidth.com")
+				.persist()
+				.get("/v1.0/api/accounts/id/test")
+				.reply(200, "<Test>test</Test>", {});
+		});
+
+		after(function () {
+			nock.cleanAll();
+			nock.enableNetConnect();
+		});
+		it("should make a request", function (done) {
+			var message = new Message({
+				getUserAgentHeader : function () {
+					return "agent";
+				}
+			});
+			message._makeIrisRequest({
+				auth : {
+					accountId : "id",
+					userName  :  "userName",
+					password  : "password"
+				},
+				path : "test"
+			}).then(function (result) {
+				result[0].Test._text.should.equal("test");
+				done();
+			}, done);
+		});
+	});
+	describe("createApplication()", function () {
+		before(function () {
+			nock.disableNetConnect();
+			nock("https://dashboard.bandwidth.com")
+				.persist()
+				.post("/v1.0/api/accounts/id/applications",
+					"<Application><AppName>App1</AppName><CallbackUrl>url</CallbackUrl><CallBackCreds/></Application>")
+				.reply(200, "<?xml version=\"1.0\" encoding=\"UTF-8\" standalone=\"yes\"?>" +
```

Comment:
better to keep these XML responses requests in another file and use template strings? Could help update tests if these payloads ever need to get updated

Reply:
Done. Moved to `templates.json`